ui/apps/platform/src/Containers/Clusters/ClustersTablePanel.tsx

```diff
@@ -422,31 +454,38 @@ function ClustersTablePanel({
                     <PanelHeadEnd>{headerActions}</PanelHeadEnd>
                 </PanelHead>
                 <PanelBody>
+                    {errorMessage && (
+                        <Alert
+                            variant="warning"
+                            isInline
+                            title="Unable to fetch clusters"
+                            component="p"
+                        >
+                            {errorMessage}
+                        </Alert>
+                    )}
```

Comment:
Did you mean to have another possible error message here?

If so, did you mean for it to have the same title as the one above?

Reply:
Good eyes.
* Error message above is for **initial** request only (if I have reorganized state correctly).
* Error message here is for **subsequent** requests, so I decided to render it in addition to the table, which displays data from last successful request.